### DISCUSSION TITLE
Removing backdrop from windows (ex: notification history)?

### DISCUSSION THREAD

**Discussion by Raagh** [LazyVim/LazyVim#5118] [Q&A] [answered]
Hi, so I am trying to remove backdrop from all windows.

I removed it from fzf-lua  and the general lazyvim config
```lua
  {
    "ibhagwan/fzf-lua",
    opts = {
      winopts = {
        backdrop = 100,
      },
      hls = {
        backdrop = 100,
        border = "FloatBorder",
        preview_border = "FloatBorder",
      },
    },
  },
  
 
    ui = {
    -- The border to use for the UI window. Accepts same border values as |nvim_open_win()|.
    border = "rounded",
    backdrop = 100,
  },
```

but there are still some windows that have it for instance the notification history and the scratch buffer. How can I remove it from all?

![image](https://github.com/user-attachments/assets/75b41644-5523-45d1-acea-1e48994f4493)

![image](https://github.com/user-attachments/assets/c77a36ff-d089-4f37-99da-dd5238b4bf92)

**Comment by dpetka2001** [comment, marked as answer]
```lua
return {
  {
    "folke/snacks.nvim",
    opts = {
      win = {
        backdrop = 100
      },
    },
  },
}
```
Should do it.

**Reply by Raagh** [reply]
yeah that was it, thank you very much
